torchci/rockset/commons/__sql/individual_test_stats_per_workflow_per_oncall.sql

```diff
@@ -14,6 +14,9 @@ WITH t AS (
     MAX(max_errors) AS max_errors 
   FROM 
     metrics.aggregated_test_metrics 
+  WHERE
```

Comment:
Now I'm zooming into this a bit more, I think we should also gate this by the workflow head branch, i.e. workflow.head_branch = 'main', as gathering the value from trunk is more stable.  Any values from PR is kind of work in progress and could be skewed.  We do the same for TTS and many other metrics.  Thought?

Reply:
It is actually already gated in the data collection in the query lambda `commons.test_insights_per_daily_upload ` (the prod version is actually the second most recent version).